test/oauth-authorization-test.js

```diff
@@ -478,21 +540,18 @@ suite.addBatch({
                         "and we submit the authentication form": {
                             topic: function(browser) {
                                 var cb = this.callback;
-                                browser.fill("username", "alice", function(err) {
-                                    if (err) {
-                                        cb(err);
-                                    } else {
-                                        browser.fill("password", "white*rabbit", function(err) {
-                                            if (err) {
-                                                cb(err);
-                                            } else {
-                                                browser.pressButton("#authenticate", function(err) {
-                                                    cb(err, browser);
-                                                });
-                                            }
-                                        });
+
+                                browser
+                                    .fill("username", "alice")
```

Comment:
See [above comment][1]: put the first`.fill` on the same line as `browser`

 [1]: https://github.com/e14n/pump.io/pull/1136/files#r48222719